functional-programming.md

```diff
@@ -5,7 +5,7 @@ A way of writing code that focuses on using pure functions describing the relati
 ##  Program Paradigm
  How to write /  organize your code, how to reason about programming.
 
-  - **Procedural** :
+  - **Procedure** :
```

Comment:
I think the best would be Procedural Programming  https://en.wikipedia.org/wiki/Procedural_programming